rollup.config.js

```diff
@@ -68,6 +82,19 @@ export default [
     input: './lib/test/index.js',
     output: {
       file: './dist/unit-tests.js',
+      format: 'esm',
```

Comment:
On master, the build times are already pretty significant:

Cold:

```
real	0m19.718s
user	0m28.090s
sys	0m1.101s
```

Hot:

```
real	0m16.310s
user	0m24.035s
sys	0m0.846s
```

I suspect some breaking up the build in the future and revisiting https://github.com/GoogleWebComponents/model-viewer/pull/235 -- although we left off on the dependency relationship of tests

Reply:
Definitely welcome work to make this better in the future.